components/occurrences/OccurrenceCodeBlock.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright 2021 The Rode Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import Button from "components/Button";
+import styles from "styles/modules/Occurrences.module.scss";
+
+const OccurrenceCodeBlock = ({ occurrence }) => {
+  const [showCode, setShowCode] = useState(false);
```

Comment:
When did this become possible? I swear this is something new. :)

Reply:
Anything is possible with JavaScript ✨